### PR TITLE
NAS-113313 / 22.02-RC.2 / NAS-113313: Removes closed slide in components from DOM

### PR DESCRIPTION
**Pull request by undsoft**
Also renames IxModal to IxSlidein.

Testing: test that component relying on IxSlideIn are working, observe that when slide in is closed inner component no longer remains in DOM.

### CODE REVIEW

File: src/app/pages/common/ix-forms/components/ix-slide-in/ix-slide-in.component.ts [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+    this.slideInService.close();
+  }
+
+  closeSlideIn(): void {
+    this.isSlideInOpen = false;
+
+    // Destroying child component later improves performance a little bit.
+    setTimeout(() => {
+      this.slideInBody.clear();
+    }, 0);
```
Comment (AlexKarpov98) [inline src/app/pages/common/ix-forms/components/ix-slide-in/ix-slide-in.component.ts:60]: Set the delay time 200ms to destroy Dom after finished transition of slide-in.

File: src/app/pages/common/ix-forms/components/ix-slide-in/ix-slide-in.component.ts [hunk 2]
```diff
@@ -0,0 +1,76 @@
...
+
+  openSlideIn<T>(componentType: Type<T>): T {
+    this.isSlideInOpen = true;
+    const componentFactory = this.componentFactoryResolver.resolveComponentFactory(componentType);
+
+    this.slideInBody.clear();
+
+    const componentRef = this.slideInBody.createComponent<T>(componentFactory);
+
+    document.body.classList.add('ix-slide-in-open');
```
Comment (AlexKarpov98) [inline src/app/pages/common/ix-forms/components/ix-slide-in/ix-slide-in.component.ts:73]: No need to toggle class name `ix-slide-in-open`  to body (also it's style).

**Review by AlexKarpov98 — COMMENTED** [review]
Looks good!
Also rename `ix-modal-header` to `ix-slide-in-header`
Don't use same ids of `jw-modal`  and `ix-slide-in` in  `admin-layout.component.html`

**Comment by undsoft** [comment]
I deliberately left `ix-modal-header` as is, because it's generic enough to maybe be used in actual modals. If not I'll rename it later.
Fixed other comments.

**Review by AlexKarpov98 — APPROVED** [review]

**Pull request merged**